packages/abi-gen/src/index.ts

```diff
@@ -43,6 +44,12 @@ const args = yargs
         demandOption: true,
         normalize: true,
     })
+    .option('backend', {
+        describe: 'Which backend do you plan to use. Either web3 or ethers',
```

Comment:
`The backing Ethereum library your app uses. Either 'web3' or 'ethers'. Ethers auto-converts small ints to numbers whereas Web3 doesn't.`